### PR TITLE
indexers: support for Cardigann and C# 'replaces' statement.

### PR DESCRIPTION
**Pull request by mynameisbogdan**
#### Description
I was thinking about moving the renamed list withing the Cardigann YML definitions for easier management.

### CODE REVIEW

File: src/Jackett.Common/Services/IndexerManagerService.cs [hunk 1]
```diff
@@ -36,25 +36,20 @@ public class IndexerManagerService : IIndexerManagerService
 
         // this map is used to maintain backward compatibility when renaming the id of an indexer
         // (the id is used in the torznab/download/search urls and in the indexer configuration file)
-        // if the indexer is removed, remove it from this list too
+        // if the indexer is removed, remove it from this list to
```
Comment (garfield69) [inline src/Jackett.Common/Services/IndexerManagerService.cs:39]: too is the correct adverb in this context, it means `also`. ;-)

**Comment by ilike2burnthing** [comment]
I haven't tested this yet (I need to reacquaint myself with the alien idea of 'sleep' first), but I'm completely on board with the concept, especially if it helps Prowlarr users as well..

Would we want to do something similar for C# indexers? e.g. for BroadcasTheNet.cs:

```cs
    public class BroadcasTheNet : IndexerBase
    {
        public override string Id => "broadcasthenet";
        public override string[] Replaces => "broadcastthenet";
        public override string Name => "BroadcasTheNet";
```

Or do we just stick to IndexerManagerService.cs for them?

**Review by ilike2burnthing — APPROVED** [review]
All working fine!

**Comment by garfield69** [comment]
so for testing, how would one go about proving that the replaces statements work as intended?

**Comment by garfield69** [comment]
and then restart the jackett service?

**Comment by garfield69** [comment]
nice.
got
```

**Review by garfield69 — APPROVED** [review]
and for C#
```
06-18 16:19:00 Info Old configuration detected: C:\ProgramData\Jackett\Indexers\passtheheadphones.json
06-18 16:19:00 Info Configuration renamed: C:\ProgramData\Jackett\Indexers\passtheheadphones.json => C:\ProgramData\Jackett\Indexers\redacted.json
```

LGTM

**Comment by garfield69** [comment]
oops, forgot to switch back to master, and pushed torrentsir ;-D

**Comment by garfield69** [comment]
So, happy for me to merge this?

**Comment by ilike2burnthing** [comment]
I think this would need to be added to Prowlarr at the same time, at least for cardigann, otherwise it'll be a pain in the ass when it comes to definitions merges.

**Pull request merged**